mobot/mobot_client/timeouts.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2021 MobileCoin. All rights reserved.
+
+import datetime
+import pytz
+import time
+
+from mobot_client.models import DropSession, Customer, Message
+from mobot_client.drop_session import ItemSessionState
+from mobot_client.chat_strings import ChatStrings
+
+utc = pytz.UTC
+
+
+class Timeouts:
+
+    def __init__(self, messenger, schedule=30, idle_timeout=60, cancel_timeout=300):
+        self.messenger = messenger
+        self.schedule = schedule
+
+        # Time before warning
+        self.idle_timeout = idle_timeout
+
+        # Time before cancelling session and refunding payment
+        self.cancel_timeout = cancel_timeout
+
+    @staticmethod
+    def customer_is_active(customer):
+        for session in DropSession.objects.filter(customer=customer).values('state'):
+            if session['state'] in ItemSessionState.active_states():
+                return True
+        return False
```

Comment:
I think this is:
`return DropSession.objects.filter(customer=customer, state__in=ItemSessionState.active_states()).exists()`

Reply:
Applies in other places in this file.